### PR TITLE
Big Red Last Stand Update The Re-Opening

### PR DESCRIPTION
**Pull request by PalJohnson**
## About The Pull Request

- This update is seeking to shake up the survivor gameplay by providing unique scenarios for survivors. This update adds several survivor holdouts (neutral & hostile) for survivors. In addition I added a minor bugfix for synth survivor,

## Why It's Good For The Game

- One of the most underrated aspects of survivors is the ability to be able to do a last stand. The current meta shift revolves around LZ holds or roaming if that's not possible. This update should encourage survivors not to leave their place of work and instead fight and in most cases die for their company if they feel inclined to as opposed to roaming or cheesing out LZ holds.

## Non-Goals

- Giving survivors additional gear. Given that their gear is random and spread across the maps survivors will now have a focal point to restock on valued items in a localized area and runoff. Although survivors are expected to stay and fight to the last colonist, nothing stops them from looting the place. In the end of the day its player agency and I don't think it would be good to force them to do something they don't want to do like die in the bar, library or marshalls. Given that most the CLF insert have 25% spawn rate, as well as the dnd, insert this shouldn't be a problem. Given the frequency of the bar holdout, I have purposely given them fewer resources.
- 
- Xenomorphs have a harder time killing survivors. Although killing or capturing survivors is not a xenomorph goal as they can ignore them entirely. Having a holdout at places like bar or even libraries for that matter could prove to be a nuisance due to premade defenses and gear being available. As stated previously nothing guarantees the survivors staying there in the first place so they could be unmanned and easily melted. Don't assault a survivor holdout alone.

## Changelog

_**- Maps**_

This update as mentioned adds 3 new survivor holdouts for survivors that will randomly spawn throughout the map. Currently, there are only 3 being added but more will be added in the future. The colony didn't get raptured it was a bloody mess with people crying and fighting a hell hole. The MR has been balanced using the CLF crashed ship insert on LV-624 as an example as to how much medical, guns and other items will be spawned in for survivors. These are prepared positions so they should have ok gear and protection if we actually want to encourage people to hold it.

_**CLF Marshalls Take Over**_ **(25% Chance)**
image

- As stated previously the CLF have invaded the colony and have taken over the last bastion of the authority of the colony that is the colonial marshall's offices and armory. After the successful takeover, the CLF decided to make the ruins of the office into their own area of operation for future CLF activities in the colony. much like the crashed ship insert, they have been equipped with identical gear to their LV-624 counterparts with minor exceptions. The exception is materials to repair and potentially expand their area of influence. The barricades for the most part are worn out and need some repairs if they expect to live long enough.

_**Dungeons and Dragons Last Stand**_ **(25% Chance)**
image
- This nightmare insert was inspired by the event held by jarek where survivors attempted to have a sleepover in the LV nexus dormitories only to be slaughtered. This insert mirrors that in that colonists are hosting a dnd session in the local library. Given that the civilian survivor is rather lackluster I have gone ahead and supplied them with cosplay material so that they can arm themselves and carry out their dnd session in style. Much of it is fluff in many regards with the exception of the illegally smuggled merc gear. The balance of the insert is similar in the crashed ship insert but the barricades are repaired and the shutters closed.

_**The Bar Last Stand**_ **_(40% Chance)_**
image
- This insert reflects the valiant efforts of the colonist to have a last stand within the colony bar. The bar (according to the colonist handbook) was designated as an emergency rally point for all personnel and staff in the event of an unforeseen event. They come equipped with standard wooden barricades with building material should they want to replace them. In addition, they have ammo, medical supplies, and liquid courage to help them in their endeavor.

_**Minor Updates**_

- Synth survivor now has colonist radio. Previously in the synth survivor update, I accidentally gave them the wrong radio. This prevented them from communicating with survivors as they had a unique radio. The auto lathes produce marine radios so they couldn't find an adequate communication device.

- Colonist survivor now has the colonist jumpsuit. They had PJs for some reason. Odd

🆑 TheFlyingFlail
add: Added three new survivor holdout nightmare inserts for big red
add: Added clf survivor to big red with their own unique story to go with it
expansion: Expanded existing mechanics or gameplay
fix: fixed synth survivors not having colonists' radio
fix: fixed civilian survivor not having actual colonist clothing
/:cl:

### CODE REVIEW

**Comment by PalJohnson** [comment]
Re-opened after approval

File: maps/map_files/BigRed/sprinkles/25.dndsolaris.dmm [hunk 1]
```diff
@@ -0,0 +1,93 @@
+"ab" = (/obj/structure/barricade/metal/wired{dir = 4},/turf/open/floor{icon_state = "wood"},/area/bigredv2/outside/library)
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.dndsolaris.dmm:1]: This it not in TGM format.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 2]
```diff
@@ -0,0 +1,150 @@
+"ax" = (/turf/open/floor{icon_state = "wood"},/area/bigredv2/outside/bar)
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:1]: This is not in TGM Format.

File: maps/map_files/BigRed/sprinkles/25.clfmarshalls.dmm [hunk 3]
```diff
@@ -0,0 +1,250 @@
+"aM" = (/obj/structure/pipes/standard/simple/hidden/green{dir = 4},/turf/open/floor{icon_state = "dark"},/area/bigredv2/outside/marshal_office)
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.clfmarshalls.dmm:1]: This is not in TGM Format.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
Not in TGM Format, fix it, ask on Discord if unclear

**Comment by PalJohnson** [comment]
bumping so bot dosent delete this

File: html/changelogs/archive/2022-05.yml [hunk 4]
```diff
@@ -211,6 +211,32 @@
   nanu308, stan_albatross:
```
Comment (Nanu308) [inline html/changelogs/archive/2022-05.yml:211]: Why are you touching this file?

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 5]
```diff
@@ -0,0 +1,1459 @@
...
+/area/bigredv2/outside/bar)
+"fi" = (
+/obj/structure/machinery/alarm{
+	dir = 4;
+	pixel_x = -30
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:57]: 25k is way too much, when a plasteel cade has 800.
A reinforced wall on BR has about 9k from memory.

File: maps/map_files/BigRed/sprinkles/25.clfmarshalls.dmm [hunk 6]
```diff
@@ -0,0 +1,2480 @@
...
+/obj/structure/pipes/standard/simple/hidden/green,
+/turf/open/floor{
+	dir = 4;
+	icon_state = "redcorner"
+	},
+/area/bigredv2/outside/marshal_office)
+"ZB" = (
+/obj/structure/window/reinforced{
+	dir = 4;
+	health = 80
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.clfmarshalls.dmm:1744]: why manually touch the health here?

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 7]
```diff
@@ -0,0 +1,1459 @@
...
+	name = "Storm Shutters";
+	pixel_x = 32
+	},
+/obj/structure/machinery/camera/autoname{
+	dir = 8
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 4;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:146]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 8]
```diff
@@ -0,0 +1,1459 @@
...
+	dir = 8;
+	icon_state = "carpet15-15";
+	tag = "icon-carpet15-15 (WEST)"
+	},
+/area/bigredv2/outside/bar)
+"oD" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:254]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 9]
```diff
@@ -0,0 +1,1459 @@
...
+/obj/item/storage/firstaid/regular,
+/turf/open/floor{
+	icon_state = "wood"
+	},
+/area/bigredv2/outside/bar)
+"pf" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:275]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 10]
```diff
@@ -0,0 +1,1459 @@
...
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:280]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 11]
```diff
@@ -0,0 +1,1459 @@
...
+/turf/open/floor/plating,
+/area/bigredv2/outside/dorms)
+"qY" = (
+/obj/structure/machinery/light{
+	dir = 8
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:330]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 12]
```diff
@@ -0,0 +1,1459 @@
...
+	name = "\improper Bar Maintenance";
+	req_one_access = null
+	},
+/turf/open/floor/plating,
+/area/bigredv2/outside/bar)
+"sg" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:358]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 13]
```diff
@@ -0,0 +1,1459 @@
...
+/obj/effect/landmark/item_pool_spawner/survivor_ammo/buckshot,
+/turf/open/floor{
+	icon_state = "wood"
+	},
+/area/bigredv2/outside/dorms)
+"tq" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:395]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 14]
```diff
@@ -0,0 +1,1459 @@
...
+	dir = 1;
+	tag = "icon-tube1 (NORTH)"
+	},
+/turf/open/floor/plating,
+/area/bigredv2/outside/bar)
+"tR" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:432]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 15]
```diff
@@ -0,0 +1,1459 @@
...
+	icon_state = "carpet15-15";
+	tag = "icon-carpet15-15 (WEST)"
+	},
+/area/bigredv2/outside/bar)
+"ul" = (
+/obj/item/paper_bin,
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 4;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:445]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 16]
```diff
@@ -0,0 +1,1459 @@
...
+/turf/open/floor{
+	icon_state = "wood"
+	},
+/area/bigredv2/outside/dorms)
+"CB" = (
+/obj/structure/pipes/standard/simple/hidden/green,
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:564]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 17]
```diff
@@ -0,0 +1,1459 @@
...
+/turf/closed/wall/solaris,
+/area/bigredv2/outside/general_offices)
+"Ea" = (
+/obj/structure/machinery/light{
+	dir = 1
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:597]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 18]
```diff
@@ -0,0 +1,1459 @@
...
+/obj/structure/pipes/standard/manifold/hidden/green{
+	dir = 8
+	},
+/turf/open/floor/plating,
+/area/bigredv2/outside/dorms)
+"Ii" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:684]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 19]
```diff
@@ -0,0 +1,1459 @@
...
+	},
+/turf/open/floor{
+	icon_state = "wood"
+	},
+/area/bigredv2/outside/bar)
+"Jk" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:716]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 20]
```diff
@@ -0,0 +1,1459 @@
...
+	},
+/area/bigredv2/outside/bar)
+"Ou" = (
+/obj/structure/pipes/standard/simple/hidden/green{
+	dir = 4
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 4;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:802]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 21]
```diff
@@ -0,0 +1,1459 @@
...
+	},
+/turf/open/floor{
+	icon_state = "wood"
+	},
+/area/bigredv2/outside/bar)
+"QG" = (
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:849]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 22]
```diff
@@ -0,0 +1,1459 @@
...
+/turf/closed/wall/solaris,
+/area/bigredv2/outside/dorms)
+"Rs" = (
+/obj/structure/pipes/standard/simple/hidden/green{
+	dir = 4
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:867]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 23]
```diff
@@ -0,0 +1,1459 @@
...
+/turf/open/floor/plating,
+/area/bigredv2/outside/bar)
+"SF" = (
+/obj/structure/pipes/standard/simple/hidden/green{
+	dir = 4
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:900]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 24]
```diff
@@ -0,0 +1,1459 @@
...
+	},
+/area/bigredv2/outside/bar)
+"SX" = (
+/obj/structure/machinery/light{
+	dir = 1
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 1;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:913]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm [hunk 25]
```diff
@@ -0,0 +1,1459 @@
...
+	},
+/obj/structure/barricade/wooden{
+	desc = "This barricade is heavily reinforced. Nothing short of blasting it open seems like it'll do the trick, that or melting the breams supporting it...";
+	dir = 8;
+	health = 25000
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.bigredbarls.dmm:918]: See earlier comment.

File: maps/map_files/BigRed/sprinkles/25.clfmarshalls.dmm [hunk 26]
```diff
@@ -0,0 +1,2480 @@
...
+/area/bigredv2/outside/marshal_office)
+"MV" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/effect/decal/cleanable/blood,
+/turf/open/floor,
+/area/bigredv2/outside/marshal_office)
+"Ni" = (
+/obj/structure/window/reinforced{
+	dir = 4;
+	health = 80
```
Comment (Nanu308) [inline maps/map_files/BigRed/sprinkles/25.clfmarshalls.dmm:1291]: See earlier comment.

**Review by Nanu308 — CHANGES_REQUESTED** [review]
See requested changes.

**Comment by Nanu308** [comment]
Also add your ckey after the CL at the start, so people know who's doing the change.

File: html/changelogs/archive/2022-05.yml (hunk 4 above)
Comment (PalJohnson) [reply html/changelogs/archive/2022-05.yml:211]: im not sure how that happen ill try to remove it

**Comment by PalJohnson** [comment]
MR mega fucked becuase of rebasing. Will re-open

**Pull request closed without merging**